### ISSUE TITLE
[BUG] Module's name can be shown in different font size

### ISSUE THREAD

**Issue by giving-sesame** [Fox2Code/FoxMagiskModuleManager#314]
**Describe the bug**
When I launch FoxMMM, some modules' names are shown in smaller font size. When I exit and then launch FoxMMM again, those modules are shown in normal font size, but now other modules' names are smaller. This picture changes each time I launch the application.

**To Reproduce**
Just launch the application and check the list of modules. Some of them have names in smaller size.

**Expected behavior**
All modules' names are of the same size.

**Screenshots**
I added a gif explaining the problem. It contains 2 screenshots of 2 different launched of FoxMMM.
![Untitled](https://user-images.githubusercontent.com/76995969/235252744-55e6a686-445b-4e55-9604-71170e88f8e1.gif)

**Device info(please complete the following information):**
 - Device: Redmi 9
 - OS: crDroid

 - App Version: 2.0.1

**Comment by Fox2Code** [comment]
It's because I made the system in a way that is not easy for @androidacy-user to maintain, prob because the view doesn't get reset properly. @androidacy-user make sure to always set every value in the update function of [ModuleViewAdapter](https://github.com/Fox2Code/FoxMagiskModuleManager/blob/master/app/src/main/java/com/fox2code/mmm/module/ModuleViewAdapter.java) even if you need to set it to it's already initial value.

**Comment by Fox2Code** [comment]
**TL;DR** It's my fault for making messy system that is hard to maintain.

**Comment by androidacy-user** [comment]
If you can still reproduce against v2.0.2, please open an issue at https://github.com/Androidacy/MagiskModuleManager

**Comment by giving-sesame** [comment]
Yes, the bug still exists in 2.0.2. Will open an issue there
